src/components/Uploader/style.scss

```diff
@@ -121,21 +120,21 @@
 }
 
 .faceIcon {
-  &:extend(.icon);
+  @extend .icon;
```

Comment:
And same here - IMO you should add both classes on JSX.

Reply:
Also fixed in latest commit(s) now. :)